omnigibson/examples/object_states/dicing_demo.py

```diff
@@ -79,29 +81,31 @@ def main(random_selection=False, headless=False, short_exec=False):
 
     # Update the simulator's viewer camera's pose so it points towards the table
     og.sim.viewer_camera.set_position_orientation(
-        position=np.array([0.544888, -0.412084, 1.11569]),
-        orientation=np.array([0.54757518, 0.27792802, 0.35721896, 0.70378409]),
+        position=th.tensor([0.544888, -0.412084, 1.11569]),
+        orientation=th.tensor([0.54757518, 0.27792802, 0.35721896, 0.70378409]),
     )
 
     # Let apple settle
     for _ in range(50):
-        env.step(np.array([]))
+        env.step(th.empty(0))
 
     knife.keep_still()
     knife.set_position_orientation(
-        position=apple.get_position() + np.array([-0.15, 0.0, 0.2]),
-        orientation=T.euler2quat([-np.pi / 2, 0, 0]),
+        position=apple.get_position() + th.tensor([-0.15, 0.0, 0.2]),
+        orientation=T.euler2quat([-math.pi / 2, 0, 0]),
     )
 
-    input("The knife will fall on the apple and dice it. Press [ENTER] to continue.")
+    if short_exec == False:
+        input("The knife will fall on the apple and dice it. Press [ENTER] to continue.")
 
     # Step simulation for a bit so that apple is diced
-    for i in range(1000):
-        env.step(np.array([]))
+    for _ in range(1000):
+        env.step(th.empty(0))
 
-    input("Apple has been diced! Press [ENTER] to terminate the demo.")
+    if short_exec == False:
+        input("Apple has been diced! Press [ENTER] to terminate the demo.")
 
-    # Always close environment at the end
+    # Always close simulator at the end
     env.close()
```

Comment:
I thought `env.close()` doesn't do anything. In general, when we run through these examples as tests, should we call `og.clear()` at the end of every example?